src/sections/LoginAuth0.js

```
@@ -23,38 +23,42 @@ class LoginAuth0 extends Component {
       domain: props.domain,
       clientID: props.clientId
     });
+    this.state = {
+      isLoggingIn: false
+    };
   }
 
   componentDidMount() {
-    const props = this.props;
     this.webAuth.parseHash(
       { hash: window.location.hash },
       (err, authResult) => {
+        window.location.hash = '';
         if (err) {
-          // TODO: Handle errors
+          // TODO: Handle errors better
+          alert(`${err.error}: ${err.errorDescription}`);
```

Don't forget to remove this.

---

(Or is not there another error reporting system?)

---

Not yet. The `alert` is there because it is better than having a silent failure.